src/plugins/intel_cpu/src/dnnl_postops_composer.cpp

```diff
@@ -251,48 +251,62 @@ void DnnlPostOpsComposer::appendClip(const std::vector<float>& low, const std::v
     }
 }
 
-MemoryPtr DnnlPostOpsComposer::prepackDecompressionParams(const MemoryCPtr& params_ptr, size_t icBlock) {
+MemoryPtr DnnlPostOpsComposer::prepackDecompressionParams(const MemoryCPtr& params_ptr, size_t icBlock, bool needTranspose) {
     // Prepacking params from [oc] to [oc, icBlock] layout, where for each icBlock corresponding parameter is duplicated
     const auto shape = params_ptr->getShape().getStaticDims();
-    const size_t elements_count = std::accumulate(shape.begin(), shape.end(), 1, std::multiplies<size_t>());
-    DnnlBlockedMemoryDesc memoryDesc(InferenceEngine::Precision::FP32, Shape({icBlock * elements_count}));
-    auto mem = std::make_shared<Memory>(engine, memoryDesc);
+
+    MemoryPtr mem;
+
     size_t dstIdx = 0;
```

Comment:
Minor: `dstIdx` is used only if `needTranspose == false`. Can we move it to the corresponding section?

Reply:
Done